server/reports/expiring-items/2_3_0/convert_data_js/src/utils.js

```diff
@@ -13,7 +13,7 @@ const processStockLines = (nodes, sort, dir) => {
     if (!!expectedUsage) {
       line.expectedUsage = Math.min(
         expectedUsage,
-        totalNumberOfPacks ?? expectedUsage
+        line?.totalNumberOfPacks * line?.packSize ?? expectedUsage
```

Comment:
Why is it totalNumberOfPacks * packSize? The expected usage should be based on average consumption * diff in months between expiryDate and now... and if that ist available then it should show not applicable

Reply:
@roxy-dao, `!!expectedUsage` would deal with AMC not available case. And this line caps expectedUsage to totalNumberOfPacks (because that's all that can be used)

Reply:
Missed Math.min. Thanks Andrei